scripts/add-task.js

```diff
@@ -38,7 +39,7 @@ function addTask(
       content: inputElement.value,
       date: currentDate,
       done: false,
-      compleTime: 0,
+      completeTime: 0,
```

Comment:
`completeTime` can be calculated based on the `date` property of the task where needed. `completeTime` initialized with `0` is not useful.